src/oci.h

```diff
@@ -109,6 +109,9 @@
 /** Mode for \ref CC_OCI_WORKLOAD_FILE. */
 #define CC_OCI_SCRIPT_MODE		0755
 
+/** Mode for cgroup directory. */
+#define CC_OCI_CGROUP_MODE		0755
```

Comment:
OK, in which case, I'd add a comment in the code explaining that's why you can't use `CC_OCI_DIR_MODE` (otherwise, someone might come along later and -- trying to be helpful -- "fix" this assuming it's a valid change to make ;-).